api/base/settings/defaults.py

```diff
@@ -162,6 +162,7 @@
         '2.9',
         '2.10',
         '2.11',
+        '2.12',
```

Comment:
yes, it's a race!  Alternatively, instead of changing this now, John, we can just put a note on this ticket that the version needs to be accurate before we merge.